### DISCUSSION TITLE
Enable only DMs that i have recieved in the current session to be displayed in chat list.

### DISCUSSION THREAD

**Discussion by gowon11** [ppy/osu#30849] [Ideas] [answered]
The DMs list can get pretty extensive and even with the recent update that added the search option for chats it's still difficult to keep a normal conversation with people, mainly because its very inconvenient to find them on that list. With that in mind it would be nice if there was an option to enable only the chats i have received in the current session to be displayed in the list, similar to how stable behaves. Or maybe option to sort the DMs list by the most recent person that sent me a message.

**Comment by peppy** [comment, marked as answer]
Recent DMs will be sorted to the top in the next release. You can follow progress [here](https://github.com/ppy/osu/issues/30835).

**Reply by gowon11** [reply]
yay
